core/network/peering_config.hpp

```diff
@@ -18,7 +18,7 @@ namespace kagome::network {
     std::chrono::seconds aligningPeriod = std::chrono::seconds(5);
 
     /// Target amount of active peers
-    size_t targetPeerAmount = 4;
+    size_t targetPeerAmount = 1;
```

Comment:
Debugging value? Possible init by different value for release and debug build (i.e. by NDEBUG macro-definition)